Exercise 24-JS/index.js

```diff
@@ -0,0 +1,76 @@
+"use strict";
+
+let x = 0;
+for (x = 0; x < 400; x++) {
+    console.log(x);
+    if (x > 401) { break; }
```

Comment:
You should calculate the sum of numbers in the loop and break the loop when the sum reaches 400

Reply:
Fixed!